edx_rest_framework_extensions/tests/test_middleware.py

```diff
@@ -116,20 +116,26 @@ def test_request_user_id_attribute(self, mock_set_custom_attribute):
         self.request.user = UserFactory()
 
         self.middleware.process_response(self.request, None)
-        mock_set_custom_attribute.assert_any_call('request_user_id', self.request.user.id)
-        mock_set_custom_attribute.assert_any_call(
-            'request_authenticated_user_found_in_middleware', 'process_response'
-        )
+
+        expected_calls = [
+            call('enduser.id', self.request.user.id),
+            call('request_user_id', self.request.user.id),
```

Comment:
Can we just remove this now that it's deprecated?

Reply:
1. The old if could be used in alerts or dashboards, so it should remain tested.
2. I did just rename the test function to match the non-deprecated attribute name.